scripts/modules/aux_services.py

```diff
@@ -138,3 +138,12 @@ def _content(self):
             labels=None,
             logging=None,
         )
+
+    def isHealthCheckServiceDisabled(self, name):
+        return (
+            name == "opbeans-load-generator" or
+            name == "filebeat" or
```

Comment:
just thought, but how about enabling [`http.enabled`](https://www.elastic.co/guide/en/beats/filebeat/master/http-endpoint.html) and healthchecking that for all beats?

Reply:
Sure, let me raise a new PR with that particular check :)

Reply:
yep, we use it for the DoD Docker images https://repo1.dsop.io/dsop/elastic/beats/auditbeat/-/blob/development/Dockerfile#L90